x/project/handler.go

```diff
@@ -402,20 +404,21 @@ func handleMsgCreateEvaluation(ctx sdk.Context, k Keeper, pk payments.Keeper,
 		),
 	})
 
-	return sdk.Result{Events: ctx.EventManager().Events()}
+	return &sdk.Result{Events: ctx.EventManager().Events()}, nil
 }
 
 func handleMsgWithdrawFunds(ctx sdk.Context, k Keeper, bk bank.Keeper,
-	msg MsgWithdrawFunds) sdk.Result {
+	msg MsgWithdrawFunds) (*sdk.Result, error) {
 
 	withdrawFundsDoc := msg.Data
 	projectDoc, err := k.GetProjectDoc(ctx, withdrawFundsDoc.ProjectDid)
 	if err != nil {
-		return sdk.ErrUnknownRequest("Could not find Project").Result()
+		return nil, sdkerrors.Wrap(did.ErrInvalidDid, "could not find project")
 	}
 
 	if projectDoc.Status != PaidoutStatus {
-		return sdk.ErrUnknownRequest("Project not in PAIDOUT Status").Result()
+		return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest,
+			"project not in PAIDOUT status")
```

Comment:
In the rest of the file if we get an unexpected project status, the error type is `sdkerrors.ErrUnauthorized`.

Reply:
Correct